front/src/app/LogsTable.tsx

```diff
@@ -0,0 +1,35 @@
+import { llm_logs } from "@prisma/client";
```

Comment:
I'm thinking maybe we change the name llm_logs before we fully commit - it's kinda ugly for the type to not be PascalCase